command/volume_snapshot_create.go

```diff
@@ -100,16 +99,13 @@ func (c *VolumeSnapshotCreateCommand) Run(args []string) int {
 
 	// Check that we at least one argument
 	args = flags.Args()
-	if l := len(args); l == 0 {
-		c.Ui.Error("This command takes at least one argument: <vol id> [snapshot name]")
+	if l := len(args); l <= 1 {
```

Comment:
I think this check is incorrect as we now want the argument length to be 2. We could therefore make this check `!= 2` or similar.